migrations/20200913230617-insert_survey_solutions_report_query.js

```diff
@@ -0,0 +1,39 @@
+const Uuid = require('cassandra-driver').types.Uuid;
+const config = require('../config/config');
+
+module.exports = {
+  async up(db) {
+    global.migrationMsg = "Insert survey solutions report query";
+    
+    
+    if (!cassandra) {
+      throw new Error("Cassandra connection not available.");
+    }
+
+    const query = 'SELECT id FROM ' + config.cassandra.keyspace + '.' + config.cassandra.table + ' WHERE qid = ? ALLOW FILTERING';
+    const result = await cassandra.execute(query, ['survey_solutions_report_query'], { prepare: true });
+    const row = result.rows;
+
+    if (!row.length) {
+
+    let id = Uuid.random();
+
+    let query = 'INSERT INTO ' + config.cassandra.keyspace + '.' + config.cassandra.table +' (id, qid, query) VALUES (?, ?, ?)';
+   
+    let queries = 
+      [{
+        query: query,
+        params: [id.toString(), 'survey_solutions_report_query','{"queryType":"groupBy","dataSource":"sl_survey","granularity":"all","dimensions":["surveySubmissionId","solutionId","solutionName","questionName","questionAnswer","remarks","surveyId","questionResponseType","questionResponseLabel","questionId","questionExternalId","instanceId","instanceParentQuestion","instanceParentResponsetype","instanceParentId","questionSequenceByEcm","instanceParentExternalId","instanceParentEcmSequence","completedDate"],"filter":{"type":"and","fields":[{"type":"selector","dimension":"solutionId","value":""}]},"aggregations":[],"postAggregations":[],"limitSpec":{"type":"default","limit":10000,"columns":[{"dimension":"questionExternalId","direction":"ascending"}]},"intervals":["1901-01-01T00:00:00+00:00/2101-01-01T00:00:00+00:00"]}']
```

Comment:
@deepakr123 - Can you share me the response for 100+ submissions made for a survey ? Need to load only aggregated report from duid and first 10 for text questions.